common/buildcraft/silicon/tile/TileLaser.java

```diff
@@ -119,6 +120,20 @@ private void updateLaser() {
         }
     }
 
+    private boolean changedTarget() {
+        if (previousTarget == null && targetPos != null) {
+            return true;
+        }
+        if (previousTarget != null && targetPos == null) {
+            return true;
+        }
+        if (previousTarget != null) {
+            return !previousTarget.equals(targetPos);
+        } else {
+            return false;
+        }
+    }
```

Comment:
You can use ````! Objects.equal(previousTarget, targetPos) ````

Reply:
Yeah, that's much better. Being rusty in my Java caught me again. I'll get that fixed